python/fledge/services/core/api/auth.py

```diff
@@ -348,39 +349,38 @@ async def create_user(request):
     _logger.info(msg)
     return web.json_response({'message': msg, 'user': u})
 
+# FIXME: Need to fix user id dependency in update_me
 
-async def update_user(request):
+
+async def update_me(request):
     """ update user profile
 
     :Example:
-             curl -X PUT -d '{"real_name": "AJ", "description": "Normal user", "role_id": "2"}' http://localhost:8081/fledge/user/<id>
+             curl -H "authorization: <token>" -X PUT -d '{"real_name": "AJ"}' http://localhost:8081/fledge/user
     """
     if request.is_auth_optional:
         _logger.warning(FORBIDDEN_MSG)
         raise web.HTTPForbidden
 
-    uid = request.match_info.get('id')
+    uid = request.match_info.get('user_id')
+    if not request.user_is_admin or int(request.user["id"]) != int(uid):
+        msg = "admin privileges are required to logout other user"
```

Comment:
logout -> update

Reply:
Even this user based check is not required. As update on the basis of token. Hence removed

Reply:
perfect!